### DISCUSSION TITLE
Ticketid : 1797872 no response since 7dayss

### DISCUSSION THREAD

**Discussion by aditisia** [community/community#33905] [Other Feature Feedback, Questions, & Ideas] [answered]
I am currently logged in through my personal account. I raised this ticket 1797872 one week ago but did not get any response. My phone is damaged which has 2FA and also I don't have recovery codes. Hence, I cant login to my organization GitHub account , can't do any development work. Please help me in trying to resolve my issue. It is very urgent. Please understand .

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

The Community Forum is unable to escalate or interfere with Support tickets. I understand how frustrating it is and I appreciate your patience, but our Support team answers tickets in the order they were received based on the order they were received. Sit tight!

**Reply by aditisia** [reply]
Hi, Can you please flag it or something. I am just stuck , not able to do any work , not sure what to tell my organisation. Please help me logging in my account.

**Comment by aditisia** [comment]
Hi Community/Community Support Team,

I really need your help in restoring my account. Details are mentioned in
ticket #1797872. I have done multiple follow ups. Please try to understand
I am not able to work , i don't know what to tell my organisation.
Please help. I really need it.

Thank you
Aditi

**Reply by queenofcorgis** [reply]
Hi there! It looks like this ticket was solved - please refer to the steps from our last communication on how to proceed. Best of luck!

**Reply by rohan717** [reply]
Hello @queenofcorgis,
Can you have a look at this ticket as well? please please consider as high priority :( 
#1908643 
I'm not able to recover my account because of the same problem, I've lost my 2FA device and recovery codes. this is for my work ID, Please help me with this. It's urgent.
Thanks in advance.

**Comment by queenofcorgis** [comment]
Hi there! [Support](https://support.github.com/) is the only place that could assist in a locked-out account, the Community is unable to help nor escalate. Going to lock this post now as the only avenue for being locked out of your account is through Support. Please be patient as our team has a process for reviewing tickets.
